chainio/clients/fireblocks/client.go

```diff
@@ -0,0 +1,162 @@
+package fireblocks
+
+import (
+	"bytes"
+	"context"
+	"crypto/rsa"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/golang-jwt/jwt"
+	"github.com/google/uuid"
+)
+
+type FireblocksTxID string
+
+type FireblocksClient interface {
+	ContractCall(ctx context.Context, body *ContractCallRequest) (*ContractCallResponse, error)
+	ListContracts(ctx context.Context) ([]WhitelistedContract, error)
```

Comment:
Could you detail here how ListContracts is supposed to be used. Unclear what its goal is and why we need it.
Since there's no parameters, where are the WhitelistedContracts coming from? Is this for OFAC compliance purposes... or?

Reply:
Makes sense. Can you just add this as a comment above the Method?